en/docs/getting-started/basic-architecture.md

```diff
@@ -0,0 +1,73 @@
+#Basic Architecture and Key Components
+
+![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)
+
+## Management Plane
+
+The management plane is where API creation and API management takes place.  It consists of portals (publisher, developer portal, admin portal) for users to create and manage APIs, implement rate-limiting policies, monitor and monetization, etc. It also provides a set of APIs to interact with external tools like API Controller. The management plane includes API Analytics dashboards, displaying various business insights. . 
+
+### Components
+
+#### API Publisher
+
+![API Publisher]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
+
+WSO2 API Manager’s API Publisher is a state-of-the-art GUI based tool for API development and management. The GUI is designed for API creators to develop, document, secure, test, and version APIs with ease. It’s also able to cater to more API management-related tasks such as publishing APIs, monetizing APIs, and applying rate-limiting policies.
+
+#### API Developer Portal
+
+![Developer Portal]({{base_path}}/assets/img/get_started/developer-portal-overview.png)
+
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to and consume APIs securely and easily.
+
+
+#### API Analytics
+
+![API Analytics]({{base_path}}/assets/img/get_started/analytics.png)
+
+Monitoring and analytics of APIs deployed on WSO2 API Manager are provided by the WSO2 API Manager Analytics component. This component includes different types of dashboards to provide more detailed different statistical views of your APIs. Furthermore, you can secure your system by configuring alerts and notifications on pre-determined events to identify any unusual behaviors in near real-time.  For more information, see Working with Analytics.
+
+
+## Data Plane
+
+Data plane is where the created API is exposed to the public consumers and acts as the proxy for API calls. This also provides additional capabilities such as enforcing security, rate limiting etc. (policy enforcement point).
+
+### Components
+
+#### API Gateway
+
+![API Gateway]({{base_path}}/assets/img/learn/gateway-overview.png)
+
+WSO2 API Gateway acts as the entry point for an API request made to an API managed by WSO2 API Manager.
+
+JWT token validation is done by the API Gateway. The API Gateway validates it's signature and subscription using the information it has in it's an in-memory map. This in-memory map includes API-related information, application-related information, subscription-related information, etc. which is updated each time an artifact (API/application) is updated.
```

Comment:
it validates the signature, issuer, expiry time, and API subscriptions. and we are using the in-memory map to validate the subscriptions only.